MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+"""Defines the Bazel module."""
+
+module(
+    name = "paralayout",
+    version = "0.0.0",
```

Comment:
Should this be "1.0.0"?

Reply:
We wouldn't update this version in `main` directly ([see example](https://github.com/bazelbuild/rules_apple/blob/master/MODULE.bazel#L3)), it gets updated automatically based on when a release is made once we start using: https://github.com/bazel-contrib/publish-to-bcr